config/locales/project.en.yml

```diff
@@ -110,6 +110,7 @@ en:
           <p>You will not be able to change this project once it is completed.</p>
       submit_button: Complete project
       back_link: Return to project list
+      unable_to_complete: This project cannot be completed until all required conditions are met and the academy has opened.
```

Comment:
I think we should state all the criteria like you have in the commit message, as a list.